src/transformers/models/vivit/modeling_vivit.py

```diff
@@ -532,8 +532,17 @@ def forward(
 
 
         >>> def sample_frame_indices(clip_len, frame_sample_rate, seg_len):
+        ...     '''
```

Comment:
Nice! Could you add these to the other doc string examples with `sample_frame_indices` too?